src/snowcli/cli/snowpark/commands.py

```diff
@@ -336,5 +337,5 @@ def _replace_handler_in_zip(
             handler_module=handler_parts[0],
             handler_function=handler_parts[1],
         )
-        utils.add_file_to_existing_zip(zip_file=zip_file_path, other_file=wrapper_file)
+        add_file_to_existing_zip(zip_file=zip_file_path, other_file=wrapper_file)
```

Comment:
Can we rename `other_file` to simply `file`?

Reply:
Done